k8s/crd-samples/replica-resource.yaml

```diff
@@ -1,9 +1,12 @@
 apiVersion: openebs.io/v1alpha1
-kind: CstorReplica
+kind: CStorVolumeReplica
 metadata:
   name: pvc-ee171da3-07d5-11e8-a5be-42010a8001be-cstor-rep-9440ab
-  poolguid: 7b99e406-1260-11e8-aa43-00505684eb2e
+  annotations:
+    poolGUID: 7b99e406-1260-11e8-aa43-00505684eb2e
+
 spec:
-  serviceAccountName: openebs-crd-operator
+#  serviceAccountName: openebs-crd-operator
+  cStorControllerIP: 10.210.102.206
   volname: vol1
```

Comment:
following the above camelCase.. should it be volName?

Reply:
will change